README.md

```diff
@@ -134,6 +134,8 @@ You can get additional help information for a particular command by specifying *
        -b, --branch <branch>            Specify the name of remote git branch. If not provided, we will try to
                                         automatically detect the current working branch if the command is used in a
                                         git enabled folder.
+                                        In cases branch is not specified AND local state has uncommitted changes or 
```

Comment:
1. Need to add that we will create a temp branch in case local branch is out of sync with remote
2. Please add at the end what the temp branch is going to be used for:
    "a temporary branch will be created and used for blueprint validation or sandbox creation."